geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -50,43 +50,82 @@
 </div>
 
 <!-- Modification Modal -->
-<ngeo-modal ngeo-modal-shown="ctrl.modifying">
+<ngeo-modal ngeo-modal-shown="ctrl.modifying || ctrl.creating">
```

Comment:
Solving a problem by adding state variables never is a good sign. Single source of trust for the win!